proxy-node-gateway/src/main/java/uk/gov/ida/notification/Metrics.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.ida.notification;
+
+import io.prometheus.client.Counter;
+
+public class Metrics {
+
+    public static final String RELEASE_NAME = getReleaseName();
+
+    private static final String RELEASE_NAME_LABEL = "release";
+    private static final String LABEL_PREFIX = "verify_proxy_node";
+
+    public static final Counter REQUESTS = Counter.build(
+            LABEL_PREFIX + "_requests",
```

Comment:
```suggestion
            LABEL_PREFIX + "_requests_total",
```
The `_total` suffix is a [Prometheus naming convention](https://prometheus.io/docs/practices/naming/) that is relatively strongly enforced and tools like `promtool check metrics` or Grafana's explore mode will complain if Counters don't have `_total` (or if non-Counter metrics do have it).